azure-pipelines.yml

```diff
@@ -47,6 +52,22 @@ jobs:
       scikit-learn
       scipy
     displayName: 'Install dependencies'
+  # TODO: recent rdkit is not on PyPI
+  - script: >-
+      python -m pip install
+      biopython
+      chemfiles
+      duecredit
+      gsd==1.9.3
+      joblib
+      GridDataFormats 
+      mmtf-python
+      networkx
+      parmed
+      tidynamics>=1.0.0
+      tqdm
```

Comment:
tqdm is a core dependency nowadays... if we are listing it anywhere in optional deps then we are doing something wrong.

Reply:
I've moved it up to the list of deps used for 32-bit and 64-bit then.

Perhaps you'll want to error out more swiftly/decisively if `tqdm` is missing, or maybe it just gets auto-installed through our `setup.py`/related checks instead of erroring. Anyway, that's probably for a separate discussion if it matters.